crates/cli-support/src/js/mod.rs

```diff
@@ -798,7 +798,7 @@ impl<'a> Context<'a> {
 
         let js = format!(
             "\
-                async function load(module, imports) {{
+                async function wasm_bindgen_load_module_imports(module, imports) {{
```

Comment:
Again, why is this called `wasm_bindgen_load_module_imports` instead of just `wasm_bindgen_load` (or `__wbg_load`)? I think I see what you're doing now, you're adding the argument names onto the end of the function name, but I don't understand why. It just makes it harder to read in my opinion.

Reply:
agreed, done!